test/integration/note_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class PregnancyNoteTest < ActionDispatch::IntegrationTest
```

Comment:
Any chance you could do me a solid and give this a slightly more descriptive filename, and match the class name? Something like `note_creation_test.rb` so we can tell exactly what actions the integration test is looking at, just by the filename.